projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -200,6 +200,7 @@ deploymentDockerRegistryPassword: ""
 deploymentDockerRegistryUsernameReadOnly: ""
 deploymentDockerRegistryPasswordReadOnly: ""
 
+
```

Comment:
Empty line

Reply:
we have committed to  https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard